source/stylesheets/_refills.scss

```diff
@@ -0,0 +1,16 @@
+.refill {
+  border-top: 2px solid $base-border-color;
+  margin-bottom: $base-spacing * 6;
+  padding-top: $base-spacing;
+}
+
+.refill-wrapper {
+  align-items: center;
+  background-color: tint($base-border-color, 90%);
+  border: $base-border;
+  display: flex;
+  justify-content: space-around;
+  margin-top: $base-spacing;
+  min-height: $base-spacing * 12;
```

Comment:
I had this mindset for quite awhile. But after working on around ~10 projects restricting myself to a well-built spacing system, I’ve actually found it to not only be achievable, but also a healthy constraint that created more logical style architecture and harmonious designs.

> also saying 'lets add more variables' does not seem like a solution to me.

The idea is that these variables would reside directly next to each other where global variables are defined. Just by this co-existence, you’d have insight into the system, rather than lots of one-off values placed throughout all of your component styles. Awareness of a system allows you to leverage it and see problems. If the spacing system grows to 50 variables, something is wrong. So while we are adding variables, it’s less about quantity and more about quality of the architecture and future maintainability.